KennySyncAdapterLab/app/src/main/java/com/korbkenny/kennysyncadapterlab/MyContentProvider.java

```diff
@@ -0,0 +1,126 @@
+package com.korbkenny.kennysyncadapterlab;
+
+import android.content.ContentProvider;
+import android.content.ContentUris;
+import android.content.ContentValues;
+import android.content.UriMatcher;
+import android.database.Cursor;
+import android.net.Uri;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+public class MyContentProvider extends ContentProvider {
```

Comment:
You didn't need to create a ContentProvider! The instructions say "You can use the Content Provider from the first app to retrieve and update data for the stocks. There is no need to create a new content provider or database."

Use the ContentProvider from the app in the starter code as an external content provider.